### DISCUSSION TITLE
dev: Where are all the modifications to the test fixtures coming from?

### DISCUSSION THREAD

**Discussion by niklaswimmer** [GitoxideLabs/gitoxide#934] [Q&A] [answered]
I ran `just t` and now I have a bunch of modifications of test fixtures (the `.tar.xz`) files in my worktree. I am unsure what to do about them (and I do not know enough about the test system to even understand why they were created). Any help from other contributors would be greatly appreciated.

I hope GH Discussions is the right medium for questions of this kind, but I found no mention of a different communication channel.

**Comment by Byron** [comment, marked as answer]
It's probably because `git lfs` isn't installed, which is why it won't turn these manifests into actual files in the working tree, which leads the test-system to recreate them from the files that it created on disk from scratch. These should definitely not be checked in - instead, please make sure that `git lfs` is available - currently there is no way to prevent the creation of these files.
